Release Notes/Release Notes v2.11.0-beta1.md

```diff
@@ -0,0 +1,42 @@
+# .NET Driver Version 2.11.0-beta1 Release Notes
+
+This is a beta release for the 2.11.0 version of the driver.
+
+The main new features in 2.11.0-beta1 are:
+
+* .NET Standard 2.0 support
```

Comment:
This implies that 2.0 wasn't supported before, which isn't exactly true.  The support is now better.  Can we describe in what way?

Reply:
@rstam What do you think?

Reply:
I would use the word "target" rather than "support" when talking about .NET Standard versions.

And you are right, we did *not* "target" .NET Standard 2.0 before.

Reply:
Updated the wording!